shared/protocol/src/main/java/io/pravega/shared/protocol/netty/Append.java

```diff
@@ -18,8 +18,8 @@
 @Data
 @RequiredArgsConstructor
 public class Append implements Request, Comparable<Append> {
-    final String segment;
-    final UUID writerId;
+    public final String segment;
```

Comment:
Why did you make these public when there are getters for them?

Reply:
Fixed. (Automated tool did this)